README.md

```diff
@@ -52,6 +52,7 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | waggledance_version                | Waggle Dance version to install on EC2 nodes                                                                                                                                                                                                                                                                | string |    `3.3.2`     | no |
 | root_vol_type                      | Waggle Dance EC2 root volume type.                                                                                                                                                                                                                                                                          | string |     `gp2`      | no |
 | root_vol_size                      | Waggle Dance EC2 root volume size.                                                                                                                                                                                                                                                                          | string |      `10`      | no |
+|enable_query_functions_across_all_metastores | This controls one Thrift call `get_all_functions` generally used to initialize a client and get build-in functions and registered UDFS from a Metastore. Setting this to `false` only gets functions from the `primary` metastore, this is more performant. Setting this to `true` will collate results by calling `get_all_functions` from all configured metastores, this is potentially slow if some of the metastores are slow to respond. If all metastores configured are the same version we'd also just get the same functions back so it's not very useful to call this across metastores, to keep backward compatibility set this to `true`. See also https://github.com/ExpediaGroup/waggle-dance#server |bool | false | no |
```

Comment:
```suggestion
|enable_query_functions_across_all_metastores | This controls the thrift call for `get_all_functions`. It is generally used to initialize a client and get built-in functions and registered UDF's from a metastore. Setting this to `false` is more performant as WD then only gets the functions from the `primary` metastore. However, setting this to `true` will collate results by calling `get_all_functions` from all configured metastores. This could be potentially slow if some of the metastores are slow to respond. If all the metastores configured are of the same version we'd also get the same functions back so it's not very useful to call this across metastores. For backwards compatibility, this property can be set to `true`. Further read: https://github.com/ExpediaGroup/waggle-dance#server | bool | false | no |
```

Reply:
yes better